Lab_13-15/raport.md

```diff
@@ -117,62 +117,76 @@ prostokąt
 
 ```sql
 SELECT  sdo_geometry (2003, 8307, null,
-sdo_elem_info_array (1,1003,3),
-sdo_ordinate_array ( -117.0, 40.0, -90., 44.0)) g
+        sdo_elem_info_array (1,1003,3),
+        sdo_ordinate_array ( -117.0, 40.0, -90., 44.0)) g
 FROM dual
 ```
 
-
-
 > Wyniki, zrzut ekranu, komentarz
 
-```sql
---  ...
```

Comment:
I think we can add the code too. It's very small in the screenshot

Reply:
Isn't the code at the beginning of each question?